tests/unit/test___init__args.py

```diff
@@ -474,6 +474,14 @@ def function_1(_arg_1):
         ),
         pytest.param(
             '''
+def _function_1(arg_1):
+    """Docstring 1."""
+''',
+            (),
+            id="prive function single arg docstring no arg",
```

Comment:
Could we also have a private function that doesn't have args but the docstring describes args?

Reply:
and then a private function that does have args and describes the args in the docstring